pyslackersweb/cli/__init__.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+import logging
+import contextlib
+
+from typing import AsyncGenerator, List
+
+from aiohttp import web
+
+from pyslackersweb.util.log import ContextAwareLoggerAdapter
+from .migrate import migrate
+
+
+logger = ContextAwareLoggerAdapter(logging.getLogger(__name__))
+
+
+@contextlib.asynccontextmanager
+async def start_app(app: web.Application) -> AsyncGenerator[None, None]:
+    logger.info("Starting cli application")
+    runner = web.AppRunner(app)
+    await runner.setup()
+    site = web.TCPSite(runner, "127.0.0.1", 8000)
```

Comment:
Do we need the app to actually bind to a port? It seems like all we need is the runner.setup/cleanup for the cli related tasks

Reply:
I wasn't sure, did some test and we actually don't need to. so that's cool